### PR TITLE
Gpt checker part 2

### PR DESCRIPTION
**Pull request by danich1**
This PR follows up on #18. Unfortunately, this contains all changes. If you prefer, you can look at everything at once on this PR instead.

### CODE REVIEW

**Comment by cgreene** [comment]
Whoops - I'm coming to this one second - do you want to do the stuff from 18 first and then we can pull those changes into this one?

**Comment by danich1** [comment]
Yeah I'll do that when I'm back at my computer. Feel free to edit the other files unique to this PR.

File: content/04.results.md [hunk 1]
```diff
@@ -2,16 +2,14 @@
...
-Our evaluation was derived from machine learning models designed to predict the actual token given a portion of its surrounding tokens.
-Each token was represented as a vector in a coordinate space constructed by these models.
-However, training these models is stochastic, which results in arbitrary coordinate spaces.
-Model alignment is essential in allowing word2vec models to be compared [@doi:10.48550/arXiv.1605.09096; @doi:10.1038/s41597-021-01047-x].
-Before alignment, each model has its own unique coordinate space (Figure {@fig:word2vec_alignment}A), and each word is represented within that space (Figure {@fig:word2vec_alignment}B).
-Alignment projects every model onto a shared coordinate space (Figure {@fig:word2vec_alignment}C), enabling direct token comparison.
+We examined how the usage of tokens in biomedical text changes over time using machine learning models.
+We trained the models to predict the actual token given a portion of its surrounding tokens, and each token was represented as a vector in a coordinate space constructed by the models.
+However, training these models is stochastic, resulting in arbitrary coordinate spaces.
```
Comment (cgreene) [inline content/04.results.md:8]:
```suggestion

However, training these models is stochastic, resulting in arbitrary coordinate spaces.
Each model has its own unique coordinate space (Figure {@fig:word2vec_alignment}A), and each word is represented within that space (Figure {@fig:word2vec_alignment}B).
Model alignment is essential in allowing word2vec models to be compared [@doi:10.48550/arXiv.1605.09096; @doi:10.1038/s41597-021-01047-x].
Alignment projects every model onto a shared coordinate space (Figure {@fig:word2vec_alignment}C), enabling direct token comparison.
```

File: content/04.results.md [hunk 2]
```diff
@@ -2,16 +2,14 @@
...
+To enable comparison of the models, we aligned them onto a shared coordinate space.
 We randomly selected 100 tokens to confirm that alignment worked as expected.
-In aligned models, tokens in the global space were more similar to themselves within the year than between years, while identical tokens in unaligned models were completely distinct (Figure {@fig:word2vec_alignment}D).
-Local distances were unaffected by alignment (Figure {@fig:word2vec_alignment}D), as token-neighbor distances were unaffected by the alignment procedure.
+We found that tokens in the global space were more similar to themselves within the year than between years, while identical tokens in unaligned models were completely distinct.
```
Comment (cgreene) [inline content/04.results.md]:
```suggestion
We found that tokens in the global space were more similar to themselves within the year than between years, while identical tokens in unaligned models were completely distinct (Figure {@fig:word2vec_alignment}D).
```

File: content/04.results.md [hunk 3]
```diff
@@ -2,16 +2,14 @@
...
+Local distances were unaffected by alignment, as token-neighbor distances remained unchanged.
```
Comment (cgreene) [inline content/04.results.md]:
```suggestion
Local distances were unaffected by alignment, as token-neighbor distances remained unchanged (Figure {@fig:word2vec_alignment}D).
```

File: content/04.results.md [hunk 4]
```diff
@@ -2,16 +2,14 @@
...
+Figures {@fig:word2vec_alignment}A-D show our results.
```
Comment (cgreene) [inline content/04.results.md]:
```suggestion
```

File: content/04.results.md [hunk 5]
```diff
@@ -69,18 +66,18 @@ The x-axis shows the time period since the first appearance of the token, and th
...
+Most of our detected changepoints (38019 for PMCOA and 2260 for preprints) only had a single event.
+
+We detected a changepoint in PMCOA for 'cas9' from 2012 to 2013 (Figure {@fig:preprint_published_changepoints}C).
+Before the changepoint, its closest neighbors were related to genetic elements (e.g., 'cas'1-3).
+After the changepoint, its closest neighbors became terms related to targeting, sgRNA, gRNA, and other genome editing strategies, such as 'talen' and 'zfns' (Table {@tbl:cas9_neighbor_table}).
 We detected change points for 'SARS' from 2002 to 2003 and 2019 to 2020 (Figure {@fig:preprint_published_changepoints}D), consistent with the emergences of SARS-CoV [@doi:10.1046/j.1440-1843.2003.00517.x] and SARS-CoV-2 [@doi:10.1016/j.ijantimicag.2020.105924; @doi:10.1038/s41564-020-0695-z] as observed human pathogens.
-We found miscellaneous neighbors before each change point, with use consistent with the acronym for Severe Acute Respiratory Syndrome after each (Tables {@tbl:sars_neighbor_table_one} and {@tbl:sars_neighbor_table_two}).
+Before each changepoint, the closest neighbors for 'SARS' were consideredmiscellaneous.
```
Comment (cgreene) [inline content/04.results.md]:
```suggestion
Before each changepoint, the closest neighbors for 'SARS' were difficult to synthesize and summarize.
```

File: content/04.results.md [hunk 6]
```diff
@@ -69,18 +66,18 @@ The x-axis shows the time period since the first appearance of the token, and th
...
+After the even occurs, the neighbors for 'SARS' were consistent with the acronym for Severe Acute Respiratory Syndrome (Tables {@tbl:sars_neighbor_table_one} and {@tbl:sars_neighbor_table_two}).
```
Comment (cgreene) [inline content/04.results.md]:
```suggestion
After changepoints, the neighbors for 'SARS' were consistent with the acronym for Severe Acute Respiratory Syndrome (Tables {@tbl:sars_neighbor_table_one} and {@tbl:sars_neighbor_table_two}).
```

File: content/05.discussion.md [hunk 7]
```diff
@@ -3,14 +3,15 @@
...
 
-We examined semantic changes within open-access biomedical corpora, Pubmed (PMCOA), and bioRxiv/MedRxiv, for the 2000-2022 interval.
-We developed a novel statistic incorporating multiple Word2Vec models to examine semantic changes over two decades.
-Before calculating our novel statistic, we used orthogonal processes that align each model (Figure {@fig:word2vec_alignment}).
-We found that word vectors are closer together after alignment; however, the best approach to align models still remains open for future investigation [@doi:10.1007/978-3-030-32233-5_58].
-As previously reported [@doi:10.1162/tacl_a_00008; @doi:10.48550/arXiv.1804.09692], we did find that without a correction step for the variability within and across years, comparing stable and unstable models is challenging.
-Our correction approach showed that the average distances in the earlier years have less variability using multiple models than a single model (Figure {@fig:novel_distance_validation}).
+We examined semantic changes in open-access biomedical corpora,Pubmed (PMCOA), and bioRxiv/MedRxiv,  over a two-decade period from 2000 to 2022.
```
Comment (cgreene) [inline content/05.discussion.md]:
```suggestion
We examined semantic changes in two open-access biomedical corpora, PubMed (PMCOA) and bioRxiv/MedRxiv, over a two-decade period from 2000 to 2022.
```

File: content/05.discussion.md [hunk 8]
```diff
@@ -3,14 +3,15 @@
...
+We developed a novel statistic that incorporated multiple Word2Vec models to examin semantic change over time.
```
Comment (cgreene) [inline content/05.discussion.md]:
```suggestion
We developed a novel statistic that incorporated multiple Word2Vec models to examine semantic change over time.
```

File: content/05.discussion.md [hunk 9]
```diff
@@ -3,14 +3,15 @@
...
+We used orthogonal processes to align each model, and we found that the word vectors were closer together after alignment (Figure {@fig:word2vec_alignment}).
```
Comment (cgreene) [inline content/05.discussion.md]:
```suggestion
We used orthogonal procrustes to align each model, and we found that the word vectors were closer together after alignment (Figure {@fig:word2vec_alignment}).
```

File: content/05.discussion.md [hunk 10]
```diff
@@ -3,14 +3,15 @@
...
-We developed a web application that allows users to manually examine individual tokens, making examining other examples within our 41,000 changepoints easier.
-However, approaches that can automatically validate these changepoints is an essential area that remains open for future investigation.
+After correcting for year variability, our analysis revealed more than 41,000 change points, including tokens such as 'cas9', 'pandemic', and 'sars' (Figure {@fig:preprint_published_changepoints}).
+Many of these change points overlapped between PMCOA and preprints, and were related to COVID-19 (Table {@tbl:published_preprint_change_table}).
+This indicates that the COVID-19 pandemic has been strong and immediate enough to cause rapid semantic change across both publishing paradigms [@doi:10.1371/journal.pbio.3000959; @doi:10.1371/journal.pone.0240123].
```
Comment (cgreene) [inline content/05.discussion.md]:
```suggestion
This indicates that the COVID-19 pandemic has had a sufficiently strong impact on the biomedical literature to cause rapid semantic change across both publishing paradigms [@doi:10.1371/journal.pbio.3000959; @doi:10.1371/journal.pone.0240123].
```

File: content/06.conclusion.md [hunk 11]
```diff
@@ -1,8 +1,9 @@
 # Conclusion
 
-We uncovered semantic changes within biomedical literature using a novel approach that accounts for inter- and intra-year variability.
-Our approach found 41,000 changepoints that include well-known examples such as 'cas9', 'pandemic', and 'sars'.
-We constructed a web application that allows users to manually examine these individual changepoints.
-As an extension to this project, future work may be able to determine the consistency and time-lag of semantic change between preprint and pre-publication peer-reviewed text - potentially predicting future change in pre-publication peer-reviewed text.
-Furthermore, including other preprint repositories may reveal consistencies across a broader swath of fields, or within-field analyses may reveal the earliest starting points of semantic changes that ultimately sweep through biomedicine.
-Overall, this work is one starting point regarding semantic change within biomedical literature, and we are excited to see how this landscape will change as time progresses.
+We uncovered changes in the meanings of words used in biomedical literature using a new approach that took variations between and within years  into account.
```
Comment (cgreene) [inline content/06.conclusion.md]:
```suggestion
We uncovered changes in the meanings of words used in biomedical literature using a new approach that took variations between and within years into account.
```

**Review by cgreene — APPROVED** [review]
I made some additional changes only on the files after methods here.

**Pull request merged**